app/models/user.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        unique: true,
+        autoIncrement: true,
+        allowNull: false
+      },
+      first_name: {
```

Comment:
Model properties should be in `camelCase`. You should find a way to relate de `camelCase` properties of the model with the `snake_case` fields of the table